site/index.html

```diff
@@ -50,6 +50,11 @@ <h5>Warframe Drop Data</h5>
 
             <div>
                 <input class="searchfield" placeholder="Search here..." id="search-field" type="text" autofocus="autofocus" aria-label="Search here..." />
+                <select id="search-match" aria-label="Search Matching">
```

Comment:
This block seems a bit too indented

Reply:
Seems fine to me, but you spotted a subtle bug, i forgot to include index.html in a commit and search-match should be match-search